lib/MetaCPAN/Web/Controller/About.pm

```diff
@@ -5,6 +5,16 @@ use Format::Human::Bytes;
 
 BEGIN { extends 'MetaCPAN::Web::Controller' }
 
+sub auto : Private {
+    my ( $self, $c ) = @_;
+
+    $c->add_surrogate_key('about');
+    $c->res->header(
+        'Cache-Control' => 'max-age=' . $c->cdn_times->{one_day} );
+    $c->cdn_cache_ttl( $c->cdn_times->{one_day} );
```

Comment:
This all looks good to me.  Would be nice if the role had a cdn_cache_control() method similar to the cdn_cache_ttl() method.  Would eliminate some repetition in the controllers.  Not a dealbreaker, though.